app/views/taxons/_form_fields.html.erb

```diff
@@ -1,3 +1,22 @@
+<%= f.input :path_prefix,
+  input_html: { class: 'js-path-prefix form-control' },
+  label: I18n.t('views.taxons.path_prefix'),
+  collection: path_prefixes_for_select %>
+
+<%= f.input :path_slug,
+  input_html: { class: 'js-path-slug form-control' },
+  label: I18n.t('views.taxons.path_slug') %>
+
+<%= f.input :base_path,
```

Comment:
This might be something I don't understand about Ruby/Rails, but given that this field is read-only, should it never set anything in the form? The actual model (`taxon.rb`) has a getter for `base_path` that concatenates the prefix and the slug. I can't actually test this locally right now, though because I've got an ES migration going (will test later).